geomesa-tools/bin/common-functions.sh

```diff
@@ -98,14 +98,14 @@ function containsElement() {
 
 # Define %%gmtools.dist.name%%_HOME and update the PATH if necessary.
 if [[ -z "$%%gmtools.dist.name%%_HOME" ]]; then
-  setGeoHome
+  geomesaConfigure
```

Comment:
actually I don't think we want to call configure here... that will prompt the user. The home variable is being set in the script for each distro before calling this function:
https://github.com/locationtech/geomesa/blob/master/geomesa-accumulo/geomesa-accumulo-tools/bin/geomesa#L12
so I think we can just remove the 'if' part of this statement